packages/api/src/schema/db-schema.yaml

```diff
@@ -653,6 +653,7 @@ components:
           index: true
         userTags:
           type: object
+          additionalProperties: true
```

Comment:
This doesn't need to be repeated here! The `db-schema` is deep merged with `api-schema`, so any field that is not here will be just kept!